frontend-react/src/components/FileHandlers/FileHandler.test.tsx

```diff
@@ -67,354 +79,388 @@ const fileChangeEvent = {
-        expect(input).toBeInTheDocument();
-    });
-
-    test("renders as expected (submitting)", async () => {
-        mockState({ ...INITIAL_STATE, isSubmitting: true });
-        renderWithSession(
-            <FileHandler
-                headingText="handler heading"
-                handlerType={FileHandlerType.VALIDATION}
-                fetcher={() => Promise.resolve({} as WatersResponse)}
-                successMessage=""
-                formLabel="" // will be tested later on page level
-                resetText=""
-                submitText=""
-                showSuccessMetadata={false}
-                showWarningBanner={false}
-                warningText=""
-            />
-        );
-
-        // to verify that the form is not rendered
-        const input = screen.queryByTestId("file-input-input"); //
-        expect(input).not.toBeInTheDocument();
-
         const spinner = await screen.findByLabelText("loading-indicator");
         expect(spinner).toBeInTheDocument();
-    });
-
-    test("renders as expected (errors)", async () => {
-        mockState({
-            ...INITIAL_STATE,
-            errors: [{ message: "error" } as ResponseError],
-        });
-        renderWithSession(
-            <FileHandler
-                headingText="handler heading"
-                handlerType={FileHandlerType.VALIDATION}
-                fetcher={() => Promise.resolve({} as WatersResponse)}
-                successMessage=""
-                formLabel=""
-                resetText=""
-                submitText=""
-                showSuccessMetadata={false}
-                showWarningBanner={false}
-                warningText=""
-            />
-        );
-
-        const errorTable = await screen.findByTestId("error-table");
-        expect(errorTable).toBeInTheDocument();
-
-        // to verify that the form is not rendered
-        const input = screen.queryByTestId("file-input-input"); //
-        expect(input).not.toBeInTheDocument();
-
-        // testing creation of error messaging for validation + file error
-        // for now, assuming that if this works, it will work for the other 3 combinations as well
-        const message = await screen.findByText(
-            "Please review the errors below."
-        );
-        expect(message).toHaveClass("usa-alert__text");
 
-        const heading = await screen.findByText(
-            "Your file has not passed validation"
-        );
-        expect(heading).toHaveClass("usa-alert__heading");
+        const imaginaryHeading = screen.queryByText("handler heading");
+        expect(imaginaryHeading).not.toBeInTheDocument();
     });
-
-    test("renders as expected (success)", async () => {
-        mockState({
-            ...INITIAL_STATE,
-            fileType: FileType.HL7,
-            destinations: "1, 2",
-            reportId: "IDIDID",
-            successTimestamp: new Date(0).toString(),
+    describe("post load", () => {
```

Comment:
all of the changes below, with the exception of a new assertion on the form label, are just nesting existing tests in a new describe block to allow for the beforeEach hook